internal/cli/runner_profile_list.go

```diff
@@ -1,6 +1,7 @@
 package cli
 
 import (
+	"encoding/json"
```

Comment:
Nitpick, across the project we typically do the pattern below, so we should have a newline between this and the next imports. Lots of tools automatically insert and break this once in awhile, not a big deal usually just manually clean it up.

```
import (
  stdlib

  external

  internal
)
```